ui/packages/app/src/api.ts

```diff
@@ -116,16 +116,18 @@ export const fn =
 			var ws_protocol = ws_endpoint.startsWith("https") ? "wss:" : "ws:";
 			if (is_space) {
 				const SPACE_REGEX = /embed\/(.*)\/\+/g;
-				var ws_path = Array.from(ws_endpoint.matchAll(SPACE_REGEX))[0][1];
+				var ws_path = Array.from(
```

Comment:
My idea was just to always make sure the ws_path always ends in `/` but there could be other (better) solutions